lib/y2_installbase.pm

```diff
@@ -227,11 +234,19 @@ C<$package_name>
 
 sub search_package {
     my ($self, $package_name) = @_;
-    assert_and_click 'packages-search-field-selected';
-    wait_screen_change { send_key 'ctrl-a' };
-    wait_screen_change { send_key 'delete' };
-    type_string_slow "$package_name";
-    send_key 'alt-s';    # search button
+    if (check_var('VIDEOMODE', 'text')) {
+        assert_and_click 'packages-search-field-selected';
+        wait_screen_change { send_key 'alt-p' };
+        type_string_slow "$package_name";
+        send_key 'ret';    # search
+    }
+    else {
+        assert_and_click 'packages-search-field-selected';
+        wait_screen_change { send_key 'ctrl-a' };
+        wait_screen_change { send_key 'delete' };
+        type_string_slow "$package_name";
+        send_key 'alt-s';    # search button
+    }
```

Comment:
Seems at least one line ```assert_and_click 'packages-search-field-selected'``` can be placed outside if else block ?

Reply:
Done